core-services/prow/02_config/rh-ecosystem-edge/kernel-module-management/_pluginconfig.yaml

```diff
@@ -0,0 +1,68 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - rh-ecosystem-edge/kernel-module-management
+  require_self_approval: false
+external_plugins:
```

Comment:
Are those configs copied from what we currently have in SRO?

Reply:
They are generated by `make new-repo`.